units/en/unit1/two-methods.mdx

```diff
@@ -0,0 +1,97 @@
+# The two main approaches for solving RL problems [[two-methods]]
+
+<Tip>
+Now that we learned the RL framework, how do we solve the RL problem?
+</Tip>
+
+In other terms, how to build an RL agent that can **select the actions that maximize its expected cumulative reward?**
+
+## The Policy π: the agent’s brain [[policy]]
+
+The Policy **π** is the **brain of our Agent**, it’s the function that tell us what **action to take given the state we are.** So it **defines the agent’s behavior** at a given time.
+
+<figure>
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit1/policy_1.jpg" alt="Policy">
+<figcaption>Think of policy as the brain of our agent, the function that will tells us the action to take given a state
+</figcaption>
+</figure>
+
+Think of policy as the brain of our agent, the function that will tells us the action to take given a state
+
+This Policy **is the function we want to learn**, our goal is to find the optimal policy π*, the policy that** maximizes **expected return** when the agent acts according to it. We find this *π through training.**
+
+There are two approaches to train our agent to find this optimal policy π*:
+
+- **Directly,** by teaching the agent to learn which **action to take,** given the state is in: **Policy-Based Methods.**
+- Indirectly, **teach the agent to learn which state is more valuable** and then take the action that **leads to the more valuable states**: Value-Based Methods.
+
+## Policy-Based Methods [[policy-based]]
+
+In Policy-Based Methods, **we learn a policy function directly.**
+
+This function will map from each state to the best corresponding action at that state. **Or a probability distribution over the set of possible actions at that state.**
+
+<figure>
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit1/policy_2.jpg" alt="Policy">
+<figcaption>As we can see here, the policy (deterministic) <b>directly indicates the action to take for each step.</b>
+</figcaption>
+</figure>
+
+
+We have two types of policy:
+
+- *Deterministic*: a policy at a given state **will always return the same action.**
+
+<figure>
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit1/policy_3.jpg" alt="Policy"/>
+<figcaption>action = policy(state)
+</figcaption>
+</figure>
+
+<img src="https://huggingface.co/datasets/huggingface-deep-rl-course/course-images/resolve/main/en/unit1/policy_4.jpg" alt="Policy" width="100%"/>
+
+- *Stochastic*: output **a probability distribution over actions.**

```

Comment:
Maybe also clarify that the "stochastic" won't return the same action always for a given state and it'd rather output a probability distribution over the possible actions.